Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs

```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
             var firstLetter = (int)rangeParts[0][0];
             var secondLetter = (int)rangeParts[1][0];
 
-            if (secondLetter < firstLetter)
+            /* if (secondLetter < firstLetter)
+             {
+                 throw new Exception("Invalid range specified.");
+             }*/
+
+
+            // Handle also columns after AA
+            Regex regexRange = new Regex(@"([A-Z]+)\d+");
+
+            Match match = regexRange.Match(rangeParts[0]);
+            string firstColName, secondColName;
+            int firstColNumber = 0, secondColNumber = 0;
+            if (match.Success)
+            {
+                firstColName = match.Groups[1].Value;
+                firstColNumber = letterToColumn(firstColName);
+            }
+            else
+            {
+                throw new Exception("Invalid range specified.");
+            }
+            Match match2 = regexRange.Match(rangeParts[1]);
+            if (match2.Success)
+            {
+                secondColName = match2.Groups[1].Value;
+                secondColNumber = letterToColumn(secondColName);
+            }
+            else
             {
                 throw new Exception("Invalid range specified.");
             }
 
-            return secondLetter - firstLetter + 1;
+            return secondColNumber - firstColNumber + 1;
+
+            //return secondLetter - firstLetter + 1;
         }
 
+
+        public static int letterToColumn(String letter)
```

Comment:
Please make sure that current unit tests are still passing and add new ones for the added functionality.